events/janus_rabbitmqevh.c

```diff
@@ -586,3 +616,40 @@ static void *janus_rabbitmqevh_handler(void *data) {
 	JANUS_LOG(LOG_VERB, "Leaving RabbitMQEventHandler handler thread\n");
 	return NULL;
 }
+
+
+/* Thread to handle heartbeats */
+static void *janus_rabbitmqevh_heartbeat_handler(void *data) {
+	JANUS_LOG(LOG_VERB, "Monitoring RabbitMQ HeartBeat\n");
+		struct timeval timeout;
+	timeout.tv_sec = 0;
+	timeout.tv_usec = 20000;
+	amqp_frame_t frame;
+
+	while(g_atomic_int_get(&initialized) && !g_atomic_int_get(&stopping)) {
+		amqp_maybe_release_buffers(rmq_conn);
+		/* Wait for a frame */
+		int res = amqp_simple_wait_frame_noblock(rmq_conn, &frame, &timeout);
+		if(res != AMQP_STATUS_OK) {
+			/* No data */
+			if(res == AMQP_STATUS_TIMEOUT || res == AMQP_STATUS_SSL_ERROR)
+				continue;
+			JANUS_LOG(LOG_VERB, "Error on amqp_simple_wait_frame_noblock: %d (%s)\n", res, amqp_error_string2(res));
+
+			if(rmq_conn && rmq_channel) {
+				amqp_channel_close(rmq_conn, rmq_channel, AMQP_REPLY_SUCCESS);
+				amqp_connection_close(rmq_conn, AMQP_REPLY_SUCCESS);
+				amqp_destroy_connection(rmq_conn);
+			}
+			JANUS_LOG(LOG_VERB, "Trying to reconnect with RabbitMQ Server\n");
+			int result = janus_rabbitmqevh_connect();
+			if(result < 0){
+				g_usleep(5000000);
+			}
+
```

Comment:
Remove empty line here.

Reply:
And the same note on space between brackets.

Reply:
Style updated